### ISSUE TITLE
Markdown Syntax

### ISSUE THREAD

**Issue by MarcusMoeller** [takezoe/gitbucket#787]
When I add a code block to README.md

```
$ make vagrant                   # build vagrant box
$ make vagrant-collectstatic-dev # collect all staticfiles
$ make vagrant-runserver         # run development server
```

it is formatted in one line instead of three.

**Comment by hansgru** [comment]
have you used:
<pre>
```bash
$ make vagrant                   # build vagrant box
$ make vagrant-collectstatic-dev # collect all staticfiles
$ make vagrant-runserver         # run development server
```
</pre>
for your syntax?

**Comment by MarcusMoeller** [comment]
Yes

**Comment by MarcusMoeller** [comment]
Here is an example:
https://freie.software/adfinis-sygroup/freedomvote/blob/master/README.md?raw=true

**Comment by hansgru** [comment]
you are missing blank lines before and after the code blocks.

See here: http://gitbucket.herokuapp.com/root/markdown/blob/master/README.md   (but it might disappear since it's a demo)

It is also recommended to specify what kind of syntax is used (python, bash, js, html) in the code block.

**Comment by takezoe** [comment]
We will switch markdown processor to [markedj](https://github.com/gitbucket/markedj) from pegdown in GitBucket 3.7. Almost markdown syntax issues will be solved by markedj.

**Comment by takezoe** [comment]
Merged to #912
